docs/sql-ref-syntax-qry-select.md

```diff
@@ -59,7 +68,7 @@ SELECT [ hints , ... ] [ ALL | DISTINCT ] { [ [ named_expression | regex_column_
     These table expressions are allowed to be referenced later in the FROM clause. This is useful to abstract
     out repeated subquery blocks in the FROM clause and improves readability of the query.
 
-* **hints**
+* **[hints](sql-ref-syntax-qry-select-hints.html)**
```

Comment:
If we go ahead with this, we should probably do it for all? I would separate the PR

Reply:
Okay, 
I will restore the related `clickable parameter` first, and I will do it in another separate PR.

Reply:
Done.